pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -1,42 +1,49 @@
 function omf.repo.pull
-  if test (command git config --get remote.upstream.url)
+  set -l repo_dir
```

Comment:
Let's do this in a future PR, but instead of having this whole block of code, we could just call `omf.repo.pull $OMF_PATH` where we are calling it without an argument. :D